src/javascript/EditPanel/EditPanelContent/FormBuilder/ChildrenSection/ManualOrdering/ManualOrdering.jsx

```diff
@@ -9,11 +9,14 @@ export const ManualOrdering = () => {
                 const handleReorder = (droppedName, index) => {
                     const droppedChild = field.value.find(child => child.name === droppedName);
                     const childrenWithoutDropped = field.value.filter(child => child.name !== droppedName);
+                    // Need to add a check here as child might be before the index
```

Comment:
It feels like it would be simpler to do one for loop or foreach to find droppedChild, childrenWithoutDropped and droppedItemIndex. It's no critical technically runtime is still the same but it may be a little easier to read.

Reply:
Umm I re-factored a bit not sure if it is more readable take a look